src/main/scala/zio/prelude/GenFs.scala

```diff
@@ -38,6 +51,15 @@ object GenFs {
         }
     }
 
+  /**
+   * A generator of `Future` values.
+   */
+  val future: GenF[Random with Sized, Future] =
+    new GenF[Random with Sized, Future] {
+      def apply[R1 <: Random with Sized, A](gen: Gen[R1, A]): Gen[R1, Future[A]] =
+        oneOf(Gen.throwable.map(Future.failed), gen.map(Future.successful))
```

Comment:
I had to do manually write a `Gen` for `Future` because it seems there isn't a `Gen.future` in zio-test